frontend/src/components/fields/MoleculeSelector/molecules.ts

```diff
@@ -112,3 +112,13 @@ export const moleculeOptionsGesia = [
   "SO3",
   "RUO4",
 ];
+export const moleculeOptionsHitemap = [
+  "CO2",
+  "CO",
+  "CH4",
+  "H2O",
+  "NO2",
+  "N2O",
+  "NO",
+  "OH",
+];
```

Comment:
these are the right HITEMP molecules, but in #627 we did not include "CH4" and "N2O". Can you remove them from the list for the moment ?